pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.17</version>
+    <version>2.30</version>
```

Comment:
This is the important part: it picks up https://github.com/jenkinsci/jenkins-test-harness/pull/35 and thus avoids the error described in https://github.com/jenkinsci/lib-annotation-indexer/pull/4 when running in `plugin-compat-tester`.